tests/bson/bson-utcdatetime_error-003.phpt

```diff
@@ -26,13 +26,20 @@ echo throws(function() {
 ?>
 ===DONE===
 <?php exit(0); ?>
---EXPECT--
+--EXPECTF--
+Deprecated: MongoDB\BSON\UTCDateTime::__construct(): Creating a MongoDB\BSON\UTCDateTime instance with a string is deprecated and be removed in ext-mongodb 2.0 in %S
```

Comment:
```suggestion
Deprecated: MongoDB\BSON\UTCDateTime::__construct(): Creating a MongoDB\BSON\UTCDateTime instance with a string is deprecated and be removed in ext-mongodb 2.0 in %s
```

`%S` is "zero or more", so I think you can use `%s` for all of these to actually expect a non-empty string.

Reply:
Fixed